src/cmd-restamp

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+dn=$(dirname "$0")
+# shellcheck source=src/cmdlib.sh
+. "${dn}"/cmdlib.sh
+
+print_help() {
+    cat 1>&2 <<'EOF'
+Usage: coreos-assembler restamp
```

Comment:
Bikeshed:  why not just call it `timestamp`?

Reply:
No strong opinion... I guess because `restamp` is more obviously a verb? And hints that it's about updating a timestamp vs. creating one. (Heh, I initially called it `bump-timestamp`, but that felt a bit verbose).

Reply:
For commands that are either infrequently invoked, or almost always invoked by automated tools, going a bit more verbose is IMO useful.

(And this command is in the latter case, no reason really for a human to run it right?)

Reply:
Fair enough, will fix.